src/lib/transition_frontier/persistent_frontier/worker.ml

```diff
@@ -78,10 +154,10 @@ module Worker = struct
     | New_node (Full _) ->
         failwith "impossible"
 
-  let handle_diff t hash (Diff.Lite.E.E diff) =
+  let handle_diff t (Diff.Lite.E.E diff) =
     let open Result.Let_syntax in
-    let%map mutant = apply_diff t diff in
-    Frontier_hash.merge_diff hash diff mutant
+    let%map _mutant = apply_diff t diff in
```

Comment:
We can refactor the mutant (and it's associated GADT parameter) in a future PR now that the frontier hash is dead. Will simplify the types quite a bit.